### ISSUE TITLE
Graphical issues at HiDPI

### ISSUE THREAD

**Issue by canack** [fishfolk/jumpy#460]
Hi, I using my screen at 150% scaling.
And game looks like : 
![fish](https://user-images.githubusercontent.com/26650249/180642011-9afc5db4-fcd6-4a75-ae6e-63c785f7ff39.png)
 
When I set the scaling 100%, everything works flawlessly!
![fish2](https://user-images.githubusercontent.com/26650249/180642092-48917ca2-1ead-4c90-96a0-e6f354f385d3.png)

**Comment by zicklag** [comment]
It looks like it might be related to the UI system not handling the scale correctly.

There is currently a work-in-progress effort to migrate to a new renderer and UI system, so this will probably be fixed once that is finished.

**Comment by zicklag** [comment]
I'm closing because we have completely re-done the rendering by migrating to Bevy. If you still have issues, feel free to re-open or open a new issue!
